backend/compound_library/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "compound_library"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+sea-orm ={ workspace = true, features = ["sqlx-postgres"]}
```

Comment:
Shouldn't need this feature if it's defined in the workspace

Reply:
Feature is not defined in the workspace

Reply:
Ah yeah, forgot we use `sqlx-sqlite` for one of the crates :facepalm: